LICENSE

```diff
@@ -1,19 +1,18 @@
-Some files are licensed under GPLv3.  These files have the text
+Some files are licensed under GPL-3.0.  These files have the text
```

Comment:
Outside of use in the `SPDX-License-Identifier` you can use `GPLv3` or `GPL 3` or `GPL 3.0` or any of the other common abbreviations.  It is only in the context of use with `SPDX-License-Identifier` that we have to use the official identifier which is `GPL-3.0-only`